### PR TITLE
Raho1 scroll prices update 1

### PR DESCRIPTION
**Pull request by raho1**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x ] Adding to existing spell lineage
- [ ] Bug fix

**Note:** You can safely discard any section below which doesn't apply based on selection above

---

### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:**  Added wrsETH and weETH prices on Scroll to prices.usd Spell

---

Thank you for your contribution!

### CODE REVIEW

File: models/prices/scroll/prices_scroll_tokens.sql [hunk 1]
```diff
@@ -30,5 +30,7 @@ FROM
     ('iusd-izumi-bond-usd', 'scroll', 'iUSD ', 0x0a3bb08b3a15a19b4de82f8acfc862606fb69a2d, 18),
     ('pxeth-pirex-ether', 'scroll', 'pxETH ', 0x9e0d7d79735e1c63333128149c7b616a0dc0bbdb, 18),
     ('frxeth-frax-ether', 'scroll', 'frxETH ', 0xecc68d0451e20292406967fe7c04280e5238ac7d, 18),
+    ('weeth-wrapped-eeth', 'scroll', 'weETH ', 0x01f0a31698C4d065659b9bdC21B3610292a1c506, 18),
+    ('rseth-rseth', 'scroll', 'wrsETH ', 0xa25b25548B4C98B0c7d3d27dcA5D5ca743d68b7F, 18),
```
Comment (Hosuke) [inline models/prices/scroll/prices_scroll_tokens.sql:34]: `wrsETH` has been defined at Line 29:
https://github.com/duneanalytics/spellbook/blob/ef2f8bd78e5b064e7e1962c6accccc97582977fd/models/prices/scroll/prices_scroll_tokens.sql#L29

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Review by raho1 — COMMENTED** [review]
Not sure what needs to be done here on my end, if anything?

File: models/prices/scroll/prices_scroll_tokens.sql (hunk 1 above)
Comment (Hosuke) [inline models/prices/scroll/prices_scroll_tokens.sql:34]: We should remove the duplicated line:
```suggestion
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @raho1

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**